examples/IRMQTTServer/IRMQTTServer.h

```diff
@@ -465,6 +487,9 @@ bool parseStringAndSendPronto(IRsend *irsend, const String str,
 #if SEND_RAW
 bool parseStringAndSendRaw(IRsend *irsend, const String str);
 #endif  // SEND_RAW
+#if SHT3X_ENABLE
```

Comment:
Out of curiosity, where/how is `SHT3X_ENABLE` defined?  Is it from `WEMOS_SHT3X.h`?
e.g. the following is in the `.ino` file, but not here in the `.h` file.
```c++
#include <WEMOS_SHT3X.h>
```

Reply:
Well spotted - that's a typo! :(
Pushing a fix shortly